[2024-04-18T01:20:06.440+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-18T01:20:06.662+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.runme_0 scheduled__2024-04-17T00:00:00+00:00 [queued]>
[2024-04-18T01:20:06.692+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.runme_0 scheduled__2024-04-17T00:00:00+00:00 [queued]>
[2024-04-18T01:20:06.693+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-18T01:20:06.739+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): runme_0> on 2024-04-17 00:00:00+00:00
[2024-04-18T01:20:06.763+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4099) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-18T01:20:06.757+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'runme_0', 'scheduled__2024-04-17T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpv2ku4b9r']
[2024-04-18T01:20:06.766+0000] {standard_task_runner.py:63} INFO - Started process 4110 to run task
[2024-04-18T01:20:06.766+0000] {standard_task_runner.py:91} INFO - Job 4: Subtask runme_0
[2024-04-18T01:20:06.912+0000] {task_command.py:426} INFO - Running <TaskInstance: example_bash_operator.runme_0 scheduled__2024-04-17T00:00:00+00:00 [running]> on host d7f8e5549574
[2024-04-18T01:20:07.150+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='runme_0' AIRFLOW_CTX_EXECUTION_DATE='2024-04-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-17T00:00:00+00:00'
[2024-04-18T01:20:07.157+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-18T01:20:07.160+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-18T01:20:07.164+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "example_bash_operator__runme_0__20240417" && sleep 1']
[2024-04-18T01:20:07.190+0000] {subprocess.py:86} INFO - Output:
[2024-04-18T01:20:07.193+0000] {subprocess.py:93} INFO - example_bash_operator__runme_0__20240417
[2024-04-18T01:20:08.197+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-18T01:20:08.198+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-18T01:20:08.235+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_0, execution_date=20240417T000000, start_date=20240418T012006, end_date=20240418T012008
[2024-04-18T01:20:08.286+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-18T01:20:08.327+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-18T01:20:08.331+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
